src/transforms/rename.jl

```diff
@@ -0,0 +1,43 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+""" 
+    Rename(Dict(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ))
```

Comment:
This is an internal signature. After we make it work with `Dict` we will only document the VarArg version to end users. So this string should read `Rename(:col1 => :newcol1, :col2 => :newcol2, ...)`.